irr/src/OpenGL3/Driver.cpp

```diff
@@ -53,6 +53,7 @@ void COpenGL3Driver::initFeatures()
 	TextureFormats[ECF_R16] = {GL_R16, GL_RED, GL_UNSIGNED_SHORT};
 	TextureFormats[ECF_R16G16] = {GL_RG16, GL_RG, GL_UNSIGNED_SHORT};
 	TextureFormats[ECF_D16] = {GL_DEPTH_COMPONENT16, GL_DEPTH_COMPONENT, GL_UNSIGNED_SHORT};
+	TextureFormats[ECF_D24] = {GL_DEPTH_COMPONENT24, GL_DEPTH_COMPONENT, GL_UNSIGNED_INT};
```

Comment:
In the OpenGL ES 3.0 spec, it's see to see that this is supported (https://registry.khronos.org/OpenGL/specs/es/3.0/es_spec_3.0.pdf, page 112, "Valid combinations of format, type, and sized internalformat continued from previous page")

Obviously OpenGL 3.2 would support this too if OpenGL ES 3.0 does, but I can't find a similar table in the spec (https://registry.khronos.org/OpenGL/specs/gl/glspec32.core.pdf)

Where is this information from?

Reply:
I went by https://docs.gl/gl3/glTexImage2D
ctrl+f `GL_DEPTH_COMPONENT24`